src/main/java/backend/Logic.java

```diff
@@ -157,37 +160,84 @@ public void refresh() {
         for (Map.Entry<Integer, IssueMetadata> entry : metadata.entrySet()) {
             IssueMetadata currentMetadata = entry.getValue();
             Date lastNonSelfUpdate = new Date(0);
+            Date lastSelfUpdate = new Date(0);
```

Comment:
I have to use `Date` because `event.getDate()` and `comment.getCreatedAt()` return `Date` instead of `LocalDateTime` objects. This would require a lot of refactoring of old code again. We can add that refactoring part in a separate issue.